examples/getting-started-app/playbook.custom-tasks.yml

```diff
@@ -18,25 +18,18 @@
       vars:
         cartridge_scenario:
           - deliver_package
-          - install_package
-          - configure_systemd_config
-          - my_restart
+          - update_package
+          - configure_instance
+          - restart_instance
+          - wait_instance_started
 
     - name: Make replicasets
       import_role:
         name: tarantool-cartridge
       vars:
+        control_instance:
+          name: core_1
+          console_sock: '/var/run/tarantool/core_1.control'
         cartridge_scenario:
-          - connect_to_membership
           - edit_topology
           - cleanup_expelled
-
-    - name: Config
-      import_role:
-        name: tarantool-cartridge
-      vars:
-        cartridge_scenario:
-          - configure_auth
-          - configure_app_config
-          - bootstrap_vshard
-          - manage_failover
```

Comment:
Let's remove this file (since current getting started doesn't respect new role features). I suggest updating it on doing this issue https://github.com/tarantool/ansible-cartridge/issues/193.